crates/edr_provider/src/requests/methods.rs

```diff
@@ -29,17 +29,20 @@ mod optional_block_spec {
 }
 
 /// for an invoking a method on a remote ethereum node
-#[derive(Clone, Debug, PartialEq, serde::Deserialize, serde::Serialize)]
+#[derive(Clone, Debug, PartialEq, serde::Deserialize, serde::Serialize, strum::Display)]
```

Comment:
Not necessarily a blocker just wondering if it'd make more sense to add a `method_name` method to the enum instead of deriving `Display` and putting the method names into `strum` annotations, as it's a bit confusing that `Display` for `MethodInvocation` only displays the name and omits the arguments.

Reply:
I'm happy to do that too

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/4781/commits/1aced6e50b7a6af735bc476a550e56cd28eeab2d